tools/generator/dfg/sam/sam_device_tree.py

```diff
@@ -276,7 +276,7 @@ def driverOrder(e):
                     inst = driver.addChild("instance")
                     inst.setValue(i[len(dtype):])
             if name == "gclk":
-                driver.addSortKey(lambda e: (e.name, int(e["value"])))
+                driver.addSortKey(lambda e: (e.name, int(e["value"]), e.get("peripheral", ""), e.get("instance", ""), e.get("name", "")))
```

Comment:
I guess that could be the right fix, but this is the wrong place. That is the sort key for the GCLK data, not the DMA requests :wink:.